### PR TITLE
Fix a regression that prevented CLI exports from terminating.

### PR DESCRIPTION
**Pull request by jannistsiroyannis**
A Java process does not terminate until all (normal) threads have finished executing. The ExecutorService, by (bad) design leaves it's threads alive and spinning, which means that process termination cannot occurr unless the ExecutorService is shutdown().

### CODE REVIEW

File: marc_export/src/main/java/whelk/export/marc/ProfileExport.java [hunk 1]
```diff
@@ -495,6 +495,7 @@ public void submit(ResultSet resultSet) throws SQLException, IOException {
         public void awaitCompletion() throws IOException {
             workQueue.awaitAll();
             out.close();
+            pool.shutdown();
```
Comment (niklasl) [inline marc_export/src/main/java/whelk/export/marc/ProfileExport.java:498]:
```suggestion
            try {
                out.close();
            } finally {
                pool.shutdown();
            }
```
(Maybe include `workQueue.awaitAll()` in the `try`, unless that failing would  make any futher recovery futile?)

Comment (jannistsiroyannis) [reply marc_export/src/main/java/whelk/export/marc/ProfileExport.java]: :+1: did it as a separate commit.

**Review by olovy — CHANGES_REQUESTED** [review]
This breaks the HTTP export since a new pool is created for every request.
If we make `ProfileExport` create and own the pool instead. And pass it to the constructor of `ParallelExporter`.

**Review by olovy — APPROVED** [review]
LGTM!

File: marc_export/src/main/java/whelk/export/marc/ProfileExport.java (hunk 1 above)
Comment (poxelman) [reply marc_export/src/main/java/whelk/export/marc/ProfileExport.java]: Reply

**Pull request merged**